css/style.css

```diff
@@ -351,7 +351,8 @@ ul.chars{
 }
 
 .brand-logo {
-    max-width: 140px
+    float:left;
```

Comment:
Viewport unit usage, float to keep it on the correct side of the page.

Reply:
floats can produce some real surprising layouts, not sure you really want that here